docs/guided-tour.md

```diff
@@ -140,32 +161,41 @@ must be provided before the workflow can run.
 Inputs to a Sprocket run are provided as arguments passed after the WDL document
 name is provided. Each input can be specified as either
 
-* a key value pair (e.g., `workflow.foo="bar"`), or
-* a file containing inputs within JSON (e.g., a `defaults.json` file where the
-  contents are `{ "workflow.foo": "bar" }`).
+* a key value pair (e.g., `main.is_pirate=true`)
+* a JSON file containing inputs (e.g., a `defaults.json` file where the
+  contents are `{ "main.is_pirate": true }`)
+* a YAML file containing inputs (e.g. a `defaults.yaml` file where the
+  contents are `main.is_pirate: true`)
 
 Inputs are _incrementally_ applied, meaning that inputs specified later override
 inputs specified earlier. This enables you to do something like the following to
 use a set of default parameters and iterate through sample names in Bash rather
-than create many individual JSON input files.
+than create many individual input files.
 
 ```bash
-sprocket run workflow.wdl defaults.json main.sample_name="Foo"
+sprocket run example.wdl defaults.json main.name="Ari"
 ```
+
+Note that the above command does not specify an entrypoint with the `--entrypoint`
+flag. This is because every input is using fully qualified dot notation; each
+input is prefixed with the name of the entrypoint and a period, `main.`.
+This fully qualified dot notation is required for inputs provided within a file.
+The dot notation can get repetitive if supplying many key value pairs on the command line,
+so specifying `--entrypoint` allows you to omit the repeated part of the keys.
 :::
 
 Here, we can specify the `name` parameter as a key-value pair on the command
```

Comment:
Maybe we should explain that, because the entrypoint is provided, the `main.` can be omitted.